docs/fundamentals/runtime-libraries/system-globalization-cultureandregioninfobuilder.md

```diff
@@ -0,0 +1,63 @@
+   - Populate the <xref:System.Globalization.CultureAndRegionInfoBuilder> object's regional properties by calling the <xref:System.Globalization.CultureAndRegionInfoBuilder.LoadDataFromRegionInfo%2A> method and passing a <xref:System.Globalization.RegionInfo> object that represents the region of your custom culture.
+
+3. Modify the properties of the <xref:System.Globalization.CultureAndRegionInfoBuilder> object as necessary.
+
+4. If you are planning to register the custom culture in a separate routine, call the <xref:System.Globalization.CultureAndRegionInfoBuilder.Save%2A> method. This generates an XML file that you can load and register in a separate custom culture installation routine.
+
+## Register a custom culture
+
+If you are developing a registration application for a custom culture that is separate from the application that creates the culture, you call the <xref:System.Globalization.CultureAndRegionInfoBuilder.CreateFromLdml%2A> method to load the XML file that contains the custom culture's definition and instantiate the <xref:System.Globalization.CultureAndRegionInfoBuilder> object. To handle the registration, call the <xref:System.Globalization.CultureAndRegionInfoBuilder.Register%2A> method. For the registration to succeed, the application that registers the custom culture must be running with administrative privileges on the target system; otherwise, the call to <xref:System.Globalization.CultureAndRegionInfoBuilder.Register%2A> throws an <xref:System.UnauthorizedAccessException> exception.
+
+> [!WARNING]
+> Culture data can differ between systems. If you're using the <xref:System.Globalization.CultureAndRegionInfoBuilder> class to create a custom culture that is uniform across multiple systems and you are creating your custom culture by loading data from existing <xref:System.Globalization.CultureInfo> and <xref:System.Globalization.RegionInfo> objects and customizing it, you should develop two different utilities. The first creates the custom culture and saves it to an XML file. The second uses the <xref:System.Globalization.CultureAndRegionInfoBuilder.CreateFromLdml%2A> method to load the custom culture from an XML file and register it on the target computer.
+
+The registration process performs the following tasks:
+
+- Creates an .nlp file that contains the information that is defined in the <xref:System.Globalization.CultureAndRegionInfoBuilder> object.
+- Stores the .nlp file in the %windir%\Globalization system directory on the target computer. This enables the custom culture's settings to persist between sessions. (The <xref:System.Globalization.CultureAndRegionInfoBuilder> method requires administrative privileges because the .nlp file is stored in a system directory.)
```

Comment:
@tarekgh This refers to %windir%. Is there a corresponding location for Unix OSes?

Reply:
Resolving convo so I can merge the PR, but would like to edit this still post-merge.

Reply:
@gewarren we need to have a comment/remark in this document telling `CultureAndRegionInfoBuilder` is useful for Windows only platforms. The generated `.nlp` files are not supported on non-Windows OS's. Also, even on Windows, the generated `.nlp` files can be supported only on .NET Framework (or in net core when using NLS globalization mode).